style/style.css

```diff
@@ -0,0 +1,180 @@
+/* Global styles
+--------------------------------*/
+
+body {
+    color: #000000;
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 15px;
+    line-height: 1.5;
+}
+
+img {
+    width: 300px;
+}
+
+.my-photo {
+    width: 380px;
+}
+
+a {
+    color: rgb(4, 93, 78);
+}
+
+a:hover {
+    text-decoration: none;
+    background-color:rgba(188,245,235,1);
+    border-radius: 20px;
+}
+
+h1, h2 {
+    font-family: 'Caveat', cursive;
+    font-size:45px;
+    margin: 0;
+}
+h3 {
+    margin-bottom: 0;
+}
+
+.content-wrap {
+    max-width: 800px;
+    width: 85%;
+    margin: 0 auto;
+    padding: 60px 0;
+}
+
+.divider>section {
+    border-bottom: 1px dashed #343434;
+    padding: 25px 0;
+}
+.divider>section:last-of-type {
+    border-bottom: none;
+}
+
+/* profile
+--------------------------------*/
+header {
+    background: rgba(107, 219, 223, .8);
+    color: #F7FFF7;
+}
+
+/* projects
+--------------------------------*/
+.projects {
+    background: #F7FFF7;
+}
+.projects a {
+    color:#2F3061;
+}
+.projects .btn {
+    color: #F7FFF7;
+    background: rgba(15, 111, 95, 1);
+    text-decoration: none;
+    padding: 8px;
+    border-radius:100px;
+    text-align: center; 
+    display: inline-block; 
+}
+.projects .btn:hover {
+    background: rgb(47, 48, 97, 80%);
+}
+.project-item {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    column-gap: 200px;
+    overflow: hidden;
+}
+
+.project-item h3 {
+    margin-top: 0;
+}
+
+
+.project-item h3 {
+    margin-top: 0;
+}
+
+/* work experience
+--------------------------------*/
+.work-experience {
+    background: #A1c6D7;
+    background-image: url(../assets/annie-spratt-unsplash.jpg);
+    background-size: cover;
+    background-position: top right;
+    padding-bottom: 1px;
+}
+
+/* Education
+--------------------------------*/
+.education {
+    background-image: url(../assets/sharon-mccutcheon-unsplash.jpg);
+    background-size: cover;
+    background-position: top right;
+    padding-bottom: 100px;
+}
+.education p {
+    width: 60%;
+}
+
+/* contact Info
+--------------------------------*/
+footer {
+    background: rgba(107, 219, 223, .8);
+    color: #2e382e;
+}
+.contact-list {
+    list-style-type: none;
+    padding: 0%;
+}
+
+.contact-list a {
+    padding: 15px;
+    display: inline-block;
+}
+
+.nav-bar {
+    list-style-type: none;
+    padding: 0%;
+    display: flex;
+    justify-content: right;
+}
+.nav-bar a {
+    padding: 15px;
+    display: inline-block;
+}
```

Comment:
These properties are the same as `.contact-list a`. They can be combined, like this:
```
.contact-list a, .nav-bar a {

}
```